### PR TITLE
Remove property label from wxMenuBar

### PR DESCRIPTION
**Pull request by Kvaz1r**
Fixes: #610 

I don't know is there any special code that should be modified after removing property(for clarify warning or something), but I didn't find any so I just removed occurrences.

### CODE REVIEW

**Comment by sodevel** [comment]
Unfortunately removing properties will lead to an error message when old projects get opened, you can easily test this by creating a project file that contains a menu bar with the original version, remove the property from the XML file and open the project file. The data loss is no problem in this case because the property is indeed gone, but the message will irritate users.

The only way to prevent this is to increase the project file version and convert old project files, the entry point for this is `ApplicationData::ConvertProject(ticpp::Document&, const wxString&, int fileMajor, int)`, the final method to deal with that property is `ApplicationData::ConvertObject( ticpp::Element*, int, int)`.

Increasing the project file version for just one property is a bit much, but on the other side this update should be performed. I would suggest to already add the update code for the next minor version but do not increase it (only during testing). I would do the version increase on next release, this way we could collect more update steps for a version increase.

In general it is safe to add properties without version increase, it is also safe to remove them but this will popup that error message. Modifying properties is dangerous because without properly updating this can lead to silent data loss.

**Comment by Kvaz1r** [comment]
Updated.
I found how was done converting project file after removing properties [before](https://github.com/wxFormBuilder/wxFormBuilder/blob/757dd88b06e19191f95854e02cc160102b474e48/src/rad/appdata.cpp#L1931-L1948) and did the same.

File: src/rad/appdata.cpp [hunk 1]
```diff
@@ -2213,6 +2213,19 @@ void ApplicationData::ConvertObject( ticpp::Element* parent, int fileMajor, int
...
 	/* The file is now at least version 1.15 */
+	if (fileMajor < 1 || (1 == fileMajor && fileMinor < 16))
+	{
+		static bool showRemovalWarnings = true;
+		if ("wxMenuBar" == objClass)
+		{
+			RemoveProperties( parent, std::set< std::string > { "label" } );
+			if (showRemovalWarnings)
+			{
+				wxLogMessage( _("Removed property label for class wxMenuBar because it is no longer used") );
```
Comment (sodevel) [inline src/rad/appdata.cpp:2224]: Also here

File: src/rad/appdata.cpp [hunk 2]
```diff
@@ -2213,6 +2213,19 @@ void ApplicationData::ConvertObject( ticpp::Element* parent, int fileMajor, int
 	}
 
 	/* The file is now at least version 1.15 */
+	if (fileMajor < 1 || (1 == fileMajor && fileMinor < 16))
+	{
+		static bool showRemovalWarnings = true;
+		if ("wxMenuBar" == objClass)
```
[the remark this answers is not in the document]
Comment (Kvaz1r) [reply src/rad/appdata.cpp]: Fixed as and others.

**Pull request merged**